Library/Formula/android-ndk.rb

```diff
@@ -2,14 +2,11 @@ class AndroidNdk < Formula
   homepage "https://developer.android.com/sdk/ndk/index.html"
 
   if MacOS.prefer_64_bit?
```

Comment:
If there's no `x64` release you can drop all the `if` statements and just make the `x86_64` version the default `url` and `sha256` in the normal location.